styles/_footer.scss

```diff
@@ -1,20 +1,195 @@
 /***** Footer *****/
-.footer {
-  border-top: 1px solid $border-color;
-  margin-top: 60px;
-  padding: 30px 0;
+footer {
+  margin: 0 auto;
+  @include fluid-value( padding-top, $min-width, $max-width, $whitespace-medium, $whitespace-medium--desktop );
+  max-width: $max-width-page-container;
+  background-color: $primary--light;
+  color: $white;
 
-  a { color: $secondary-text-color; }
+  .footer-inner {
+    display: block;
 
-  &-inner {
-    @include max-width-container;
-    display: flex;
-    justify-content: space-between;
+    &-column-wrap {
+      @include fluid-value( margin-top, $min-width, $max-width, $whitespace-small, $whitespace-small--desktop );
+      
+      @include tablet{
+        display: flex;
+        justify-content: space-between;
+      }
+    }
   }
 
-  &-language-selector {
-    color: $secondary-text-color;
-    display: inline-block;
-    font-weight: $font-weight-light;
+  .footer-logo-link {
+    padding-bottom: $whitespace-small;
+    box-shadow: unset;
+    
+    &:hover,
+    &:focus,
+    &:active {
+      box-shadow: unset;
+    }
+  }
+
+  .footer-logo {
+    color: $white;
+    max-width: 242px;
+  }
+
+  .footer {
```

Comment:
I see what you did here, probably not the best to havea `.footer` when at the point of this pr there was no `.footer` class. That could come off as super confusing